### PR TITLE
New tag s / Recommended to call d.openenv / Extra python path cannot be added view d 

### LINKED ISSUE #74: importd.s()
Problem: Often I have to mirror on setting with another. Eg DEBUG and TEMPLATE_DEBUG are typically the same. Its not easy to say one setting should be equal to another when configuring setting via importd, but is easy when doing it via settings.py.

Solution:

```python
from importd import d, s

d(
    DEBUG=True,
    TEMPLATE_DEBUG=s("DEBUG")
)
```

### PR DESCRIPTION
**Pull request by hitul007**
- New tag s added. ( Fixed #74 )
- It is recommended to call d.openenv before calling it in main.
- Extra python path cannot be added into d as kwargs.

### CODE REVIEW

File: importd/__init__.py [hunk 1]
```diff
@@ -268,6 +274,14 @@ def __init__(self):
         """Init class."""
         self.blueprint_list = []
 
+    def openenv(self):
```
Comment (amitu) [inline importd/__init__.py]: We have to allow path too. It should accept both relative and absolute path. If its relative, its with respect to calling file.

File: tests/mirror_settings/main.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+d(
+    DEBUG=True,
+    LOG_LOCATION=d.dotslash("temp-log.py"),
+    DEBUG_TEST=s("DEBUG")
+)
+
+if __name__ == "__main__":
+    from django.conf import settings
+    if settings.DEBUG_TEST == True:
```
Comment (amitu) [inline tests/mirror_settings/main.py]: Prefer:

```python
assert settings.DEBUG_TEST
```

**Comment by amitu** [comment]
Also docs are pending :-)

File: tests/mirror_settings_nested/main.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# make sure to use importd from the repository
```
Comment (amitu) [inline tests/mirror_settings_nested/main.py]: Update tox.ini to test for these things too.

File: importd/__init__.py (hunk 1 above)
Comment (hitul007) [inline importd/__init__.py]: Done

File: tests/mirror_settings/main.py (hunk 2 above)
Comment (hitul007) [inline tests/mirror_settings/main.py]: Done

File: tests/mirror_settings_nested/main.py (hunk 3 above)
Comment (hitul007) [inline tests/mirror_settings_nested/main.py]: Done

**Pull request merged**